pipelineruns/integration_pipelinerun_test_app.yaml

```diff
@@ -3,6 +3,8 @@ apiVersion: tekton.dev/v1
 kind: PipelineRun
 metadata:
   generateName: integration-pipelinerun-
+  annotations:
+    pipelinesascode.tekton.dev/installation-id: '43840620'
```

Comment:
I'm less sure about this one, but it probably might be best to not add it here either as it's geared towards running a specific kind of test, and not representing PaC.